### ISSUE TITLE
Markdown templates with wrong syntax

### ISSUE THREAD

**Issue by benjaminburkhardt** [arc42/arc42-template#160]
Additionally to Ticket #159 the markdown syntax is not followed. 

For example:
- table syntax
- placeholder syntax, ```{placeholder}``` should be used instead of ```<placeholder>``` because it might be interpreted as html
- single ```:   *\<Erläuternder Text>*``` does not exist
- the full version in DE stoll contains asciidoc syntax such as ```::: formalpara-title```
- ```{#_ebene_2}```should be removed

**Comment by gernotstarke** [comment]
Benjamin, you or we should check the pandoc conversion - as the Markdown version of the template is generated from the original AsciiDoc source

So instead of cleaning the markdown manually, I suggest we find a better/cleaner way of generating or transforming adoc->markdown...

cc @rdmueller

**Comment by benjaminburkhardt** [comment]
@gernotstarke definitely generating from source is the way of choice! I've not looked into pandoc yet - so if one of you guys has the chance to look into it that would be great. ;-)

**Comment by rdmueller** [comment]
yes, we should try to configure pandoc or look for another converter and not try to clean it manually.

the main problem I see is that there are so many markdown dialects. So which should we use?

For instance, the original markdown is not capable of generating tables: https://daringfireball.net/projects/markdown/syntax

For those who want to play around with the settings - the code we use to generate markdown can be found here: https://github.com/arc42/arc42-generator/blob/master/subBuild.gradle#L185

We already generate "markdown", "strict markdown" and "github markdown"

**Comment by gernotstarke** [comment]
@benjaminburkhardt - neither @rdmueller nor myself have a chance to look into this issue in the near future, therefore I ask for your help.

I leave this issue open, maybe other folks have time and/or ideas...

**Comment by arosenf** [comment]
I think "strict markdown" already solves all of the problems mentioned above. But it does not seem to be available for download on arc42.org/download but only from this repository.

**Comment by gernotstarke** [comment]
markdown is available... https://github.com/arc42/arc42-template/raw/master/dist/arc42-template-EN-plain-markdown.zip

do you suggest to add  "markdown-strict" as additional download on arc42.org?

**Comment by arosenf** [comment]
I wasn't aware that there were more download options than published on arc42.org.
It was only by chance after finding this issue that I found the markdown-strict template. Maybe it would be helpful for others if it was directly available on the website. So my answer to your question would be "yes".

**Comment by gernotstarke** [comment]
added several additional download formats...
